### PR TITLE
Develop alexey

### PR DESCRIPTION
**Pull request by balexey88**
Prepare update release 0.0.2

### CODE REVIEW

File: class-elementor.php [hunk 1]
```diff
@@ -160,4 +225,165 @@ public function filter_css_file($name, $absolutePath) {
...
+
+  /**
+   * Get the list of files to sync.
+   * 
+   * @param array $file_list
+   * @return array
+   */
+  public function get_sync_files($file_list) {
+    if ( !method_exists('\wpCloud\StatelessMedia\Utility', 'get_files') ) {
+      Helper::log('WP-Stateless version too old, please update.');
```
Comment (planv) [inline class-elementor.php]: @balexey88 

The WP-Stateless version is outdated. Please update.

File: wp-stateless-elementor-addon.php [hunk 2]
```diff
@@ -5,9 +5,9 @@
  * Plugin URI: https://stateless.udx.io/addons/elementor/
  * Description: Provides compatibility between the Elementor Website Builder and the WP-Stateless plugins.
  * Author: UDX
- * Version: 0.0.1
+ * Version: 0.0.2
  * License: GPL v2 or later
- * Text Domain: slcaelementor
+ * Text Domain: wp-stateless-elementor-website-builder-addon
```
Comment (planv) [inline wp-stateless-elementor-addon.php:10]: @balexey88 should we rename repository?

**Review by planv — DISMISSED** [review]

**Comment by planv** [comment]
There is a notification on wordpress.org 

image

It should be resolved with 

image

**Comment by balexey88** [comment]
> There is a notification on wordpress.org
> 
> image
> It should be resolved with
> 
> image

Yes, the tags are updated in the new version

**Review by planv — DISMISSED** [review]

File: wp-stateless-elementor-addon.php (hunk 2 above)
Comment (planv) [reply wp-stateless-elementor-addon.php:10]: renamed

**Review by planv — APPROVED** [review]

**Pull request merged**